Server/mods/deathmatch/logic/CPedSync.cpp

```diff
@@ -191,73 +187,148 @@ void CPedSync::Packet_PedSync(CPedSyncPacket& Packet)
 {
     // Grab the player
     CPlayer* pPlayer = Packet.GetSourcePlayer();
-    if (pPlayer && pPlayer->IsJoined())
+    if (!pPlayer || !pPlayer->IsJoined())
+        return;
+
+    // Grab the tick count
+    long long llTickCountNow = GetModuleTickCount64();
+
+    // Apply the data for each ped in the packet
+    for (auto iter = Packet.IterBegin(); iter != Packet.IterEnd(); iter++)
     {
-        // Apply the data for each ped in the packet
-        vector<CPedSyncPacket::SyncData*>::const_iterator iter = Packet.IterBegin();
-        for (; iter != Packet.IterEnd(); iter++)
+        CPedSyncPacket::SyncData* pData = *iter;
+
+        // Grab the ped this packet is for
+        CElement* pPedElement = CElementIDs::GetElement(pData->Model);
```

Comment:
Please change the `Model` in `SyncData` to `ID` or something, it's very misleading.

Reply:
Changed it to `ID`